config/main.py

```diff
@@ -516,6 +526,19 @@ def _get_sonic_generated_services(num_asic):
                 generated_services_list.append(line.rstrip('\n'))
     return generated_services_list, generated_multi_instance_services
 
+def _chk_service_stopped(service_name):
+    # Check the service for 10 times with 5s interval
+    for i in range(10):
+        (srv_active, err) = run_command("systemctl is-active {}".format(service_name))
+        if "inactive" in srv_active:
+            return True
+        time.sleep(5)
```

Comment:
This will wait up to 50 seconds. Can you please explain the choice of 50 seconds?

Reply:
It takes at most 50 seconds when I try to stop each of the services

Reply:
Maybe we can increase it to an even 60 seconds to add a buffer, and decrease the sleep to 1 second?

Reply:
Sure. Good point.